proxy/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM rust:latest AS builder
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN cargo build --release --package proxy-testgen
+
+RUN cargo build --release --package proxy
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)